serialization/example_portable_serialization_test.go

```diff
@@ -85,6 +85,7 @@ func ExampleConfig_SetPortableFactories() {
 	config := hazelcast.NewConfig()
 	config.Serialization.PortableVersion = 1
 	config.Serialization.SetPortableFactories(&MyPortableFactory{})
+	// version must be equivalent with PortableVersion
```

Comment:
"version must be equivalent to PortableVersion" ?

Reply:
As I explain [here](https://github.com/hazelcast/hazelcast-go-client/pull/758#discussion_r930342482). It was related with portable versining.

Reply:
My intention was to suggest changing the text to use "to" instead "with": ""version must be equivalent to PortableVersion" "

Reply:
My bad, fixed [5dca40b](https://github.com/hazelcast/hazelcast-go-client/pull/758/commits/5dca40bd3b2c69bfe4310bb3c293c71d358b2116).